compute_endpoint/setup.py

```diff
@@ -51,11 +52,16 @@
     exec(f.read(), version_ns)
 version = version_ns["__version__"]
 
+directory = Path(__file__).parent
+long_description = (directory / "PyPI.md").read_text()
```

Comment:
Why did you choose Markdown?  Our readthedocs is all rst, why not go with that?

As it's separate from docs/, I don't think it matters.  Just curious.

Reply:
Really just personal preference. I find markdown much easier to read and write.